From d044f0fad674bfec3dbf484ea4d0ed35fdae0ae1 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Wed, 21 Feb 2024 17:19:33 +0100
Subject: [PATCH 0658/1695] dts: stm32mp13: describe cpu power supply on
 stm32mp135f-dk board

Describe cpu power supply on stm32mp135f-dk board that
is used for CPU OPP.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c07d0a31b04fcacaa167249e22b4b39b580a98b9 dts: stm32mp13: describe cpu power supply on stm32mp135f-dk board]
Change-Id: I611ad9a63005e491efa5cfdc309288fea84ede90
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/362296
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp135f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index e3746b261..7d3bb9e1c 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -105,6 +105,10 @@
 	};
 };
 
+&cpu0 {
+	cpu-supply = <&vddcpu>;
+};
+
 &etzpc {
 	st,decprot = <
 		DECPROT(STM32MP1_ETZPC_ADC1_ID, DECPROT_NS_RW, DECPROT_UNLOCK)
-- 
2.47.2

